script/Actors.sol

```diff
@@ -1,40 +1,58 @@
-    address public constant PROCESSOR = address(2);
-    address public constant EXECUTOR_1 = address(3);
-    address public constant PROPOSER_1 = address(4);
+    /// @dev timelock
+    function FEE_MANAGER() external view returns (address);
+    /// @dev timelock
+    function PROVIDER_MANAGER() external view returns (address);
+    /// @dev timelock
+    function BUFFER_MANAGER() external view returns (address);
+    /// @dev timelock
+    function ASSET_MANAGER() external view returns (address);
+    /// @dev timelock
+    function PROCESSOR_MANAGER() external view returns (address);
+    /// @dev multisig
+    function PAUSER() external view returns (address);
+    /// @dev multisig
+    function UNPAUSER() external view returns (address);
+    /// @dev multisig
+    function PROCESSOR() external view returns (address);
 }
 
-contract HoleskyActors is IActors {
-    address public constant ADMIN = 0x743b91CDB1C694D4F51bCDA3a4A59DcC0d02b913;
-    address public constant PROCESSOR = 0x743b91CDB1C694D4F51bCDA3a4A59DcC0d02b913;
-    address public constant EXECUTOR_1 = 0x743b91CDB1C694D4F51bCDA3a4A59DcC0d02b913;
-    address public constant PROPOSER_1 = 0x743b91CDB1C694D4F51bCDA3a4A59DcC0d02b913;
+contract TestnetActors is IActors {
+    address public constant ADMIN = 0x72fdBD51085bDa5eEEd3b55D1a46E2e92f0837a5;
+    address public constant PROCESSOR = 0x72fdBD51085bDa5eEEd3b55D1a46E2e92f0837a5;
+    address public constant PROPOSER_1 = 0x72fdBD51085bDa5eEEd3b55D1a46E2e92f0837a5;
+    address public constant PROPOSER_2 = 0x72fdBD51085bDa5eEEd3b55D1a46E2e92f0837a5;
+    address public constant EXECUTOR_1 = 0x72fdBD51085bDa5eEEd3b55D1a46E2e92f0837a5;
+    address public constant EXECUTOR_2 = 0x72fdBD51085bDa5eEEd3b55D1a46E2e92f0837a5;
 
-    address public constant PROVIDER_MANAGER = 0x743b91CDB1C694D4F51bCDA3a4A59DcC0d02b913;
-    address public constant BUFFER_MANAGER = 0x743b91CDB1C694D4F51bCDA3a4A59DcC0d02b913;
-    address public constant ASSET_MANAGER = 0x743b91CDB1C694D4F51bCDA3a4A59DcC0d02b913;
-    address public constant PROCESSOR_MANAGER = 0x743b91CDB1C694D4F51bCDA3a4A59DcC0d02b913;
-    address public constant PAUSER = 0x743b91CDB1C694D4F51bCDA3a4A59DcC0d02b913;
-    address public constant UNPAUSER = 0x743b91CDB1C694D4F51bCDA3a4A59DcC0d02b913;
+    address public constant FEE_MANAGER = 0x72fdBD51085bDa5eEEd3b55D1a46E2e92f0837a5;
+    address public constant PROVIDER_MANAGER = 0x72fdBD51085bDa5eEEd3b55D1a46E2e92f0837a5;
+    address public constant BUFFER_MANAGER = 0x72fdBD51085bDa5eEEd3b55D1a46E2e92f0837a5;
+    address public constant ASSET_MANAGER = 0x72fdBD51085bDa5eEEd3b55D1a46E2e92f0837a5;
+    address public constant PROCESSOR_MANAGER = 0x72fdBD51085bDa5eEEd3b55D1a46E2e92f0837a5;
+    address public constant PAUSER = 0x72fdBD51085bDa5eEEd3b55D1a46E2e92f0837a5;
+    address public constant UNPAUSER = 0x72fdBD51085bDa5eEEd3b55D1a46E2e92f0837a5;
 }
 
 contract MainnetActors is IActors {
```

Comment:
https://github.com/yieldnest/yieldnest-kernel-lrt/blob/dev/script/Actors.sol#L80

Please use the same configuration like here

i think it can just be copy pasted